### ISSUE TITLE
v1.6.3 (v1.4.0 macros) Breaks Un-annotated Fields

### ISSUE THREAD

**Issue by theory** [jonasbb/serde_with#267]
I have a struct like this:

``` rust
#[serde_with::serde_as]
#[derive(Serialize, Deserialize, Debug, PartialEq)]
pub struct Thing {
    pub id: u8,

    #[serde_as(as = "Vec<serde_with::hex::Hex>")]
    pub data: Vec<Vec<u8>>,
}
```

It worked well until today, presumably with the release of v1.6.3, when I started getting this error:

```
error: An empty `serde_as` attribute on a field has no effect. You are missing an `as`, `serialize_as`, or `deserialize_as` parameter.
  --> src/thing:14:5
   |
14 |     pub id: u8,
   |     ^^^
```

I did not previously have to annotate the `id` field. So I tried adding an annotation:

``` rust
#[serde_with::serde_as]
#[derive(Serialize, Deserialize, Debug, PartialEq)]
pub struct Thing {
    #[serde_as(as = "u8")]
    pub id: u8,

    #[serde_as(as = "Vec<serde_with::hex::Hex>")]
    pub data: Vec<Vec<u8>>,
}
```

But now it says:

```
error[E0277]: the trait bound `u8: SerializeAs<_>` is not satisfied
   --> src/thing:11:1
    |
11  | / #[serde_with::serde_as]
12  | | #[derive(Serialize, Deserialize, Debug, PartialEq)]
    | |__________________^ the trait `SerializeAs<_>` is not implemented for `u8`
    | 
```

So how do I get the default serialization behavior back again?

**Comment by jonasbb** [comment]
Thanks for the report. I already prepared a PR to fix this in #268 and will release a new bugfix release after that is merged.

This happened because of #262. The code used to contain a check if a field has any `serde_as` attribute and exit early if not. That got removed in #262.
I added a new testcase to avoid such breakages in the future.

**Comment by jonasbb** [comment]
The new version is now on crates.io. Sorry for the inconvenience.
